genesyscloud/recording_media_retention_policy/resource_genesyscloud_recording_media_retention_policy.go

```diff
@@ -67,9 +67,18 @@ func createMediaRetentionPolicy(ctx context.Context, d *schema.ResourceData, met
 	order := d.Get("order").(int)
 	description := d.Get("description").(string)
 	enabled := d.Get("enabled").(bool)
-	mediaPolicies := buildMediaPolicies(d, pp, ctx)
```

Comment:
Basically any place we were doing a log.Fatal I refactored to return an error message and then end in a non-retryable error or a diagnostic error.   This will cause the provider to fail gracefully without "defecating" the bed and crashing the provider.